airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -701,6 +701,7 @@
   allowedHosts:
     hosts:
       - api.github.com
+      - {api_url}
```

Comment:
Should be quoted and have a $ - check out the other examples in this file, like `source-postgres`

Reply:
Quoted